modules/mongodb/templates/mongodb-backup-s3.erb

```diff
@@ -17,65 +17,100 @@ date_format = '%Y-%m-%d:%H:%M:%S'
 mongo_server = '<%= @server %>'
 key_fingerprint = '<%= @private_gpg_key_fingerprint %>'
 s3_bucket = '<%= @s3_bucket %>'
-standalone = '<%= @standalone %>'
 start = datetime.now().strftime(date_format)
```

Comment:
This variable `start` is no longer used as of this pull request so it can be removed.